wasmplugin/plugin.go

```diff
@@ -533,3 +534,16 @@ func replaceResponseBodyWhenInterrupted(bodySize int) types.Action {
 	proxywasm.LogWarn("response body intervention occurred: body replaced")
 	return types.ActionContinue
 }
+
+// parseServerName parses :authority pseudo-header in order to retrieve the
+// virtual host.
+func parseServerName(authority string) string {
+	host, _, err := net.SplitHostPort(authority)
+	if err != nil {
+		// missing port or bad format
+		proxywasm.LogDebugf("failed to parse server name from authority %q, %v", authority, err)
+		host = authority
+	}
+	// anyways serverName is returned
```

Comment:
```suggestion
```

Not sure what it means

Reply:
I meant that even if `SplitHostPort` raises an error, `host` is returned with the input value. But it is straightforward in the code itself